oio/xcute/jobs/mover.py

```diff
@@ -48,33 +48,30 @@ def _generate_fake_excluded_chunks(self, excluded_rawx):
             fake_excluded_chunks.append(chunk)
         return fake_excluded_chunks
 
-    def process(self, payload):
-        rawx_timeout = payload['rawx_timeout']
-        min_chunk_size = payload['min_chunk_size']
-        max_chunk_size = payload['max_chunk_size']
-        excluded_rawx = payload['excluded_rawx']
+    def process(self, chunk_id, task_payload):
+        rawx_id = task_payload['rawx_id']
+        rawx_timeout = task_payload['rawx_timeout']
+        min_chunk_size = task_payload['min_chunk_size']
+        max_chunk_size = task_payload['max_chunk_size']
+        excluded_rawx = task_payload['excluded_rawx']
 
-        chunk_url = 'http://{}/{}'.format(
-            payload['rawx_id'], payload['chunk_id'])
-
-        fake_excluded_chunks = self._generate_fake_excluded_chunks(excluded_rawx)
+        fake_excluded_chunks = self._generate_fake_excluded_chunks(
+            excluded_rawx)
 
+        chunk_url = 'http://{}/{}'.format(rawx_id, chunk_id)
         meta = self.blob_client.chunk_head(chunk_url, timeout=rawx_timeout)
-
         container_id = meta['container_id']
         content_id = meta['content_id']
         chunk_id = meta['chunk_id']
         chunk_size = int(meta['chunk_size'])
 
-        # Skip the chunk if it's too small or too big
+        # Maybe skip the chunk because it doesn't match the size constaint
         if chunk_size < min_chunk_size:
             self.logger.debug("SKIP %s too small", chunk_url)
-
-            return True, 0
-        if max_chunk_size > -1 and chunk_size > max_chunk_size:
+            return
```

Comment:
The call to `process` would fail, as the expected return value is a tuple:
```python
task_ok, task_result = task.process(task_id, task_payload)
```

Reply:
Oops